spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2220,6 +2220,16 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
               expr.children(2))
             None
           }
+        case expr if expr.prettyName == "size" =>
```

Comment:
Instead of matching expr name "size", we can match `Size` expression with an `ArrayType` input expression.

Reply:
Got it! Thank you for your comments